cpp/include/raft/core/mdarray.hpp

```diff
@@ -27,6 +27,12 @@
 #include <rmm/cuda_stream_view.hpp>
 
 namespace raft {
+/**
+ * @\brief Dimensions extents for raft::host_mdspan or raft::device_mdspan
+ */
+template <size_t... ExtentsPack>
```

Comment:
Include `stdint.h` for `size_t` or include `cstdint` and use `std::size_t` where possible